jenkins/ci.suse.de/openstack-mkcloud.yaml

```diff
@@ -522,7 +522,7 @@
                   ghremote=origin
                   git config --get-all remote.${ghremote}.fetch | grep -q pull || \
                       git config --add remote.${ghremote}.fetch "+refs/pull/*/head:refs/remotes/${ghremote}/pr/*"
-                  git fetch $ghremote 2>&1 | grep -v '\[new ref\]'
+                  git fetch $ghremote 2>&1 | grep -v '\[new ref\]' || :
```

Comment:
Thats also correct, but I would merge this fix with with just one review as it already let builds fail.
Now we have to wait for another review of this PR.